suite-native/module-authorize-device/src/usePassphraseAuthorizationSuccess.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+import {
+    selectIsCreatingNewPassphraseWallet,
+    selectDeviceRequestedAuthorization,
+} from '@suite-native/device-authorization';
+import { useAuthorizationGoBack } from '@suite-native/device-authorization';
+
+export const useAuthorizationSuccess = () => {
```

Comment:
note to myself: does it have to be hook? Is there some more straightforward way to handle success?

Reply:
I was going back and forth on this. I had 2 same `useEffect`s, so I changed it to hook, but now from DX, you have the functionality visibly hidden. I think if it needs to stay as a useEffect, hook might be better - by a hair (DRY).

I was looking into whether it's possible to trigger this as some callback "reaction", but it's quite troublesome since it's triggered by websocket from connect (`ui-close-window`) and not some async response 😢 

Let me know if you figure something else or want to discuss further. I'm not too happy about this myself...

Reply:
Actually, in a follow up PR, I'd like to rename this hook and use it to handle success & error states together. We can probably work on tweaking it in that follow up.